pkg/api/types.go

```diff
@@ -455,6 +455,9 @@ type PersistentVolumeSpec struct {
 	// means that this volume does not belong to any StorageClass.
 	// +optional
 	StorageClassName string
+	// A comma-separated list of mount options. Not validated--mount will simply fail if one is invalid.
+	// +optional
+	MountOptions *string
```

Comment:
Why is this a comma-sep string rather than a `[]string` ?  I dislike embedded syntax unless well-justified...